README.md

```diff
@@ -27,26 +27,18 @@ mvn package -DskipTests
 
 `mvn package` will assemble all the required dependencies and package into an uber jar.
 
-Pre-create Atlas models (Recommended)
-=====================================
+Create Atlas models
+===================
 
-You may need to create Atlas models to ensure all necessary models are created before running SAC and pushing metadata changes. Spark Atlas Connector provides the way to pre-create Atlas models, and it is only needed for the first time.
+You may need to create Atlas models to ensure all necessary models are created before running SAC and pushing metadata changes.
 
-Suppose Spark is installed in `<SPARK_HOME>` directory and `atlas-application.properties` is placed on `<SPARK_HOME>/conf` directory:
+According to Atlas team, the official and recommended approach for registering models in Apache Atlas is having json model file. We've co-worked with Atlas team to define official Spark models for Apache Atlas which is compatible with SAC.
 
-```shell
-java -cp "<spark dist>/jars/*:<spark dist>/conf:spark-atlas-connector_2.11-0.1.0-SNAPSHOT.jar" com.hortonworks.spark.atlas.types.SparkAtlasModel --interactive-auth
-```
-
-The tool will leverage REST client API to request to Atlas which requires authentication. Auth. information is read from `atlas-application.properties`, which should be stored as plain text.
-(It would be same if you decide to leverage REST client API in SAC itself.)
-
-Given the approach is insecure regardless of security mode of cluster, we strongly encourage you to pass `--interactive-auth` as parameter, which asks you to input username and password of Atlas interactively.
+SAC leverages official Spark models in Apache Atlas, but as of Apache Atlas 2.0.0, it doesn't include the model file yet. Until Apache Atlas publishes new release which includes the model, SAC includes the json model file to apply to Atlas server easily.
 
-If you would like to let SAC handles it instead, you can set you can set `atlas.client.checkModelInStart=true` in `atlas-application.properties` to let SAC check and create models in Spark Atlas Connector's each startup.
-Please note that it leverages REST API even you configure `atlas.client.type=kafka`, and it would require additional configuration for authentication. (`atlas.client.username` and `atlas.client.password`)
+Please copy `1100-spark_model.json` to `<ATLAS_HOME>/models/1000-Hadoop` directory and restart Atlas server to take effect.
 
-In the meanwhile, we're also working on migrating models into Apache Atlas: once it's done, we no longer require this step and leverage the models Atlas registers for the startup.
+With the change of approach on registering models, `atlas.client.checkModelInStart` option is removed.
```

Comment:
I think it may be useful to clarify what is the change in approach. Or remove this comment altogether.

Reply:
Revisiting this: I think it would only give confusion for new users who don't indicate about registration of model via API. So let's just remove it.